lib/routes/oceanengine/index.js

```diff
@@ -0,0 +1,142 @@
+const dayjs = require('dayjs');
+const template = require('art-template');
+template.defaults.minimize = true;
```

Comment:
```suggestion
const { art } = require('@/utils/render');
```
You can use RSSHub's wrapper of art-template
https://docs.rsshub.app/joinus/script-standard.html#template-li-zi